site/pages/Index.vue

```diff
@@ -1,181 +1,171 @@
 <template>
   <div>
     <section
```

Comment:
Not sure if Olympe intended it this time again, but in the last version the first section was always scaling up to the height of the viewport as long as the elements would fit. Here I can already have a glimpse of the next section:

![Screenshot from 2022-09-13 17-03-09](https://user-images.githubusercontent.com/34456797/189937487-8415ba51-b47c-450c-abb7-8f51169ce969.png)

Reply:
We decided that we will not use the `h-screen` approach here because Olympe liked more how the page scales on big screens without it.